Update docs with updated info (#7714)
All-Hands-AI/OpenHands: docs/modules/usage/runtimes-index.md

# Runtime Configuration

:::note

> This section is for users that would like to try out a different runtime for OpenHands.

Reviewer: ```suggestion
This section is for users that would like to use a runtime other than Docker for OpenHands.
```
